Software permissions table (#3135)
astronomer/docs: software/role-permission-reference.md

This is where you'll find information about Astronomer Software default user role permissions. To modify these default permissions, see [Customize role permissions](manage-platform-users.md#customize-role-permissions).

## Default role permissions tables

> The following tables show high level comparisons of permissions between different user roles.

Reviewer: ```suggestion
The following tables show high level comparisons of the different permitted actions between different user roles.
```

| Create, update, or delete any Team                                                                                                                |                   |                   | ✔️                |
| Invite, update, or delete any user                                                                                                                |                   |                   | ✔️                |
| Bypass email verification for any user                                                                                                            |                   |                   | ✔️                |
| Create, update, or delete a Workspace                                                                                                             |                   |                   | ✔️                |
| Airflow [admin permissions](https://airflow.apache.org/docs/apache-airflow/stable/security/access-control.html#admin) for all Deployments         |                   |                   | ✔️                |

## Default role permissions lists

> The following topics list the permission values that each role has by default as defined in the Astronomer Helm chart. You can update these permissions in your `config.yaml` file if you want to change the permissions that each role has. See [Customize role permissions](manage-platform-users.md#customize-role-permissions). 

Reviewer: ```suggestion
The following sections list the permission values that each role has by default as defined in the Astronomer Helm chart. You can update these permissions in your `config.yaml` file if you want to change the permissions that each role has. See [Customize role permissions](manage-platform-users.md#customize-role-permissions). 
```

> Note that these lists are also published in YAML form on the [Astronomer documentation repository](https://github.com/astronomer/docs/blob/main/software_configs/0.33/default.yaml).

Reviewer: ```suggestion
These lists are also published in YAML form in the [Astronomer documentation repository](https://github.com/astronomer/docs/blob/main/software_configs/0.33/default.yaml).
```

| Deployment Admin permissions for all Deployments                                                                                                  | ✔️                 | ✔️                 | ✔️                |
| View [environment variables](environment-variables.md) for any Deployment                                                                         | ✔️                 | ✔️                 | ✔️                |
| View any setting for any Deployment in the Software UI                                                                                            | ✔️                 | ✔️                 | ✔️                |
| View all pending user invites in the **System Admin** tab of the Software UI                                                                      | ✔️                 | ✔️                 | ✔️                |
| View information for any pending user invite                                                                                                      | ✔️                 | ✔️                 | ✔️                |

> 

Author: ```suggestion
```
